frontend/packages/operator-lifecycle-manager/src/components/descriptors/reference/reference.md

```diff
@@ -853,7 +854,55 @@ status:
   </tr>
 </table>
 
-#### 3. conditions
+#### 3. w3ExternalLink
+
+**x-descriptors**
+
+This descriptor allows you to expose an external link of your instance and open it in the separate browser tab or window. It expects the output format in the status block of your instance in:
+```yaml
+…
+status:
+  [PATH_TO_THE_FIELD]: [FIELD_VALUE]
+…
+```
+
+**Example**
+
+```yaml
+…
+- displayName: Eclipse Che URL
+  description: Route to access Eclipse Che
+  path: cheURL
+  x-descriptors:
+    - urn:alm:descriptor:org.w3:externalLink
+…
+```
+
+The status block of the field being specified:
+
+```yaml
+…
+status:
+  …
+  cheURL: 'http://che-tony.apps.rhamilto.devcluster.openshift.com'
+…
+```
+
+**On the DISPLAY VIEW:**
+<table style="width:100%">
+  <tr valign="top">
+    <td width="50%">
+      <img src="img/status/2-1_w3Link.png" />
+      <h6><small>Click on the link sends users to target URL.</small></h6>
+      </td>
+    <td width="50%">
+      <img src="img/status/2-2_w3Link.png" />
+      <h6><small>Hover over displayName shows descriptions on tooltip.</small></h6>
+      </td>
+  </tr>
+</table>
```

Comment:
We'll need to update these with new screenshots after the other suggested updates. ExternalLinks are styled differently.